net/batman-adv-legacy/Makefile

```diff
@@ -55,12 +54,17 @@ MAKE_FLAGS += \
 	CONFIG_BATMAN_ADV_NC=$(if $(CONFIG_KMOD_BATMAN_ADV_LEGACY_NC),y,n) \
 	REVISION="" all
 
+define Build/Compile
+	$(MAKE) -C "$(PKG_BUILD_DIR)" $(MAKE_FLAGS)
+	mv $(PKG_BUILD_DIR)/batman-adv.$(LINUX_KMOD_SUFFIX) $(PKG_BUILD_DIR)/batman-adv-legacy.$(LINUX_KMOD_SUFFIX)
+endef
+
 define KernelPackage/batman-adv-legacy/install
-	$(INSTALL_DIR) $(1)/etc/config $(1)/etc/hotplug.d/net $(1)/etc/hotplug.d/iface $(1)/lib/batman-adv $(1)/usr/sbin $(1)/lib/netifd/proto
-	$(INSTALL_DATA) ./files/etc/config/batman-adv $(1)/etc/config
-	$(INSTALL_DATA) ./files/lib/batman-adv/config.sh $(1)/lib/batman-adv
-	$(INSTALL_BIN) ./files/etc/hotplug.d/net/99-batman-adv $(1)/etc/hotplug.d/net
-	$(INSTALL_BIN) ./files/lib/netifd/proto/batadv.sh $(1)/lib/netifd/proto
+	$(INSTALL_DIR) $(1)/etc/config $(1)/etc/hotplug.d/net $(1)/etc/hotplug.d/iface $(1)/lib/batman-adv-legacy $(1)/usr/sbin $(1)/lib/netifd/proto
+	$(INSTALL_DATA) ./files/etc/config/batman-adv-legacy $(1)/etc/config
```

Comment:
Hmm, as `/etc/config/batman-adv` does not exist in the upstream batadv package anymore, we might want to migrate to the new batadv first (implementing the same changes in batadv-legacy) before allow co-existence. I think I can look into this migration over the Easter holidays.

Reply:
Ok, thanks, just ping me somewhere when that's updated. I'll rebase and update my PR afterwards then.

Reply:
is there progress? :)